docs/DeveloperGuide.adoc

```diff
@@ -775,11 +984,32 @@ Use case resumes at step 2.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  A novice who has never used an Address Book should have no problem using it.
+.  The system should respond under a second.
+.  A user private information should not be seen by others unless they are granted access.
+.  User commands should not be case sensitive.
+.  Should back up data to cloud or have a back up program to recover any loss of data.
+.  Should have a change history up to 100000 changes.
+.  Should work offline most of the time except when accessing services requiring internet (eg. cloud, social networks).
+.  Should restore to operational status under a minute after a failure occur.
+.  Should be able to continue working even after failing to restore.
```

Comment:
You can be more specific here. Continue working how? From a clean slate, or in a mode that allows changes to be merged later after a successful restore?